lib/Client.php

```diff
@@ -207,6 +207,16 @@ class Client
      */
     protected $retryOnLimit;
 
+	/**
+	 * @var bool
+	 */
+	protected $throwException;
+
+	/**
+	 * @var int
+	 */
+	protected $defaultErrorCode = 0;
```

Comment:
It exists the unused white spaces. You should check the line `208` to know what this problem is.